src/components/product/ProductCard/product-card.module.scss

```
@@ -80,6 +81,7 @@
       background-color: var(--fs-product-card-bkg-color-hover);
       border-color: var(--fs-product-card-border-color-hover);
       box-shadow: var(--fs-product-card-shadow-hover);
+      [data-product-card-image] img { transform: scale(var(--fs-product-card-img-scale-hover)); }
```

Wouldn't be the case to add `fs` here?

```suggestion
      [data-fs-product-card-image] img { transform: scale(var(--fs-product-card-img-scale-hover)); }
```